src/MusicStore/Controllers/AccountController.cs

```diff
@@ -383,7 +383,7 @@ public ActionResult ExternalLogin(string provider, string returnUrl = null)
         [ValidateAntiForgeryToken]
         public async Task<ActionResult> ExternalLoginConfirmation(ExternalLoginConfirmationViewModel model, string returnUrl = null)
         {
-            if (User.Identity.IsAuthenticated)
+            if (User.IsLoggedIn())
```

Comment:
Should we call this method `IsSignedIn()` just to be consistent with `SignInManager`, `SignIn` and `SignOut` methods in various places?